pkg/kubectl/cmd/label.go

```diff
@@ -68,6 +68,7 @@ type LabelOptions struct {
 var (
 	labelLong = templates.LongDesc(i18n.T(`
 		Update the labels on a resource.
+		Include the uninitialized objects by default, unless user explicitly set --include-uninitialized=false.
```

Comment:
I thought it's only the case if `--all` is specified?

Reply:
@caesarxuchao Sorry to forget removing this help message, since we have included in the flag description.